setup.py

```diff
@@ -1,71 +1,69 @@
-import setuptools
-from setuptools import find_packages
 import re
 from pathlib import Path
 
+import setuptools
+from setuptools import find_packages
+
 FILE = Path(__file__).resolve()
 PARENT = FILE.parent  # root directory
 README = (PARENT / "README.md").read_text(encoding="utf-8")
 
 
 def get_version():
-    file = PARENT / 'supervision/__init__.py'
-    return re.search(r'^__version__ = [\'"]([^\'"]*)[\'"]', file.read_text(encoding="utf-8"), re.M)[1]
-    
+    file = PARENT / "supervision/__init__.py"
+    return re.search(
+        r'^__version__ = [\'"]([^\'"]*)[\'"]', file.read_text(encoding="utf-8"), re.M
+    )[1]
+
 
 setuptools.setup(
-    name='supervision',
+    name="supervision",
     version=get_version(),
-    author='Piotr Skalski',
-    author_email='piotr.skalski92@gmail.com',
-    license='MIT',
-    description='A set of easy-to-use utils that will come in handy in any Computer Vision project',
+    author="Piotr Skalski",
+    author_email="piotr.skalski92@gmail.com",
+    license="MIT",
+    description="A set of easy-to-use utils that will come in handy in any Computer Vision project",
     long_description=README,
-    long_description_content_type='text/markdown',
-    url='https://github.com/roboflow/supervision',
-    install_requires=[
-        'numpy>=1.20.0',
-        'opencv-python',
-        'matplotlib',
-        'pyyaml'
-    ],
+    long_description_content_type="text/markdown",
+    url="https://github.com/roboflow/supervision",
+    install_requires=["numpy>=1.20.0", "opencv-python", "matplotlib", "pyyaml"],
```

Comment:
@SkalskiP @onuralpszr Can we make installation like `pip install supervision[tracker]` and it will install dependency for tracker otherwise use normal installation. Installation of `scikit-learn` might create an issue for `arm` devices. What do you think?

Reply:
I can see "arm64" support for scikit-learn 

https://pypi.org/project/scikit-learn/1.3.0/#files

"arm" (32-bit) already consider old. And If you buy a new arm based board, it will be high likely arm64 devices so it is fine on that as well any other package worry about ?